sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpClientNetworkStateHandler.java

```diff
@@ -52,9 +52,6 @@ public synchronized ServiceCall callAsync(String url, String method, Map<String,
     @Override
     public synchronized void close() throws IOException {
         mNetworkStateHelper.removeListener(this);
-        for (Call call : mCalls) {
```

Comment:
This is worth commenting here why we don't do that anymore.

Reply:
I believe you mean the comment in `onNetworkStateUpdated`